Use chain.from_iterable in wkb.py

cool-RR opened the pull request:
This is a faster and more idiomatic way of using `itertools.chain`. Instead of computing all the items in the iterable and storing them in memory, they are computed one-by-one and never stored as a huge list. This can save on both runtime and memory space.

cool-RR commented:
I assume that the CI failure is not related to my change.

larsbutler commented:
> I assume that the CI failure is not related to my change.

It's not; if you rebase on the latest master branch, that error should go away.

larsbutler commented:
> This is a faster and more idiomatic way of using `itertools.chain`. Instead of computing all the items in the iterable and storing them in memory, they are computed one-by-one and never stored as a huge list. This can save on both runtime and memory space.

Thanks for the patch. Do you have some profiling data that shows a difference in performance or memory usage? In any case, it's functionally equivalent and cleaner-looking. I'm happy to accept it.

Please have a read through the [contributing guide](https://github.com/geomet/geomet/blob/master/CONTRIBUTING.md) and add yourself to the [AUTHORS](https://github.com/geomet/geomet/blob/master/AUTHORS.txt) file.

cool-RR commented:
I didn't profile it, no. I went over a bunch of projects and fixed this issue en masse.

I added myself to AUTHORS, thank you. Anything else I'm missing?

larsbutler commented:
@cool-RR 

> I added myself to AUTHORS, thank you. Anything else I'm missing?

Nope, all good. Thanks for the patch!

larsbutler commented:
@cool-RR Per the [contribution guidelines](https://github.com/geomet/geomet/blob/master/CONTRIBUTING.md#project-administration), would you like to become a maintainer? If so, please have a read through the entire contribution guidelines and if you agree to all of that, just let me know and I'll add you as a maintainer.

cool-RR commented:
No, I don't want to take that responsibility.
